packages/react-instantsearch/examples/react-native/src/Home.js

```diff
@@ -0,0 +1,355 @@
+} from 'react-instantsearch/connectors';
+import Highlight from './components/Highlight';
+import Spinner from './components/Spinner';
+import StarRating from 'react-native-star-rating';
+import IosIcon from 'react-native-vector-icons/Ionicons';
+import MaterialIcon from 'react-native-vector-icons/MaterialIcons';
+
+import { Actions } from 'react-native-router-flux';
+
+const { height } = Dimensions.get('window');
+
+const styles = StyleSheet.create({
+  maincontainer: {
+    ...Platform.select({
+      ios: {
+        marginTop: 63,
+      },
+      android: { marginTop: 50 },
+    }),
+    flex: 1,
+  },
+  items: {
+    ...Platform.select({
+      ios: {
+        height: height - 170,
+      },
+      android: { height: height - 185 },
+    }),
+  },
+  item: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    backgroundColor: 'white',
+  },
+  options: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    backgroundColor: 'white',
+    padding: 5,
+    borderBottomColor: 'gray',
+    borderBottomWidth: 1,
+  },
+  sortBy: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+  },
+  sortByArrow: {
+    paddingLeft: 3,
+  },
+  searchBoxContainer: {
+    backgroundColor: '#162331',
+    flexDirection: 'row',
+    alignItems: 'center',
+  },
+  searchBox: {
+    backgroundColor: 'white',
+    height: 40,
+    borderWidth: 1,
+    padding: 10,
+    margin: 10,
+    flexGrow: 1,
+    ...Platform.select({
+      ios: {
+        borderRadius: 5,
+      },
+      android: {},
+    }),
+  },
+  itemContent: {
+    paddingLeft: 15,
+  },
+  itemName: {
+    fontSize: 20,
+    fontWeight: 'bold',
+    paddingBottom: 5,
+  },
+  itemType: {
+    fontSize: 13,
+    fontWeight: '200',
+    paddingBottom: 5,
+  },
+  itemPrice: {
+    fontSize: 15,
+    fontWeight: 'bold',
+    paddingBottom: 5,
+  },
+  starRating: { alignSelf: 'flex-start' },
+  filters: { flexDirection: 'row', alignItems: 'center' },
+});
+class Home extends Component {
+  static displayName = 'React Native example';
+  constructor(props) {
+    super(props);
+    this.onSearchStateChange = this.onSearchStateChange.bind(this);
+    this.state = {
+      searchState: this.props.searchState ? this.props.searchState : {},
+    };
+  }
+
+  componentWillReceiveProps() {}
+
+  onSearchStateChange(nextState) {
+    this.setState({ searchState: { ...this.state.searchState, ...nextState } });
+  }
+
+  render() {
+    return (
+      <View style={styles.maincontainer}>
+        <InstantSearch
+          appId="latency"
+          apiKey="6be0576ff61c053d5f9a3225e2a90f76"
+          indexName="ikea"
+          searchState={this.state.searchState}
+          onSearchStateChange={this.onSearchStateChange}
+        >
+          <StatusBar backgroundColor="blue" barStyle="light-content" />
+          <ConnectedSearchBox />
+
+          <View style={styles.options}>
+            <ConnectedStats />
```

Comment:
@marielaures just a nitpicky thing: can you add left-padding for the stats label in the first screen?